README.md

```diff
@@ -16,6 +14,13 @@ Including this repo within your own will:
 ## Setup in code repo
 
 1. Add this repo as a submodule into `./scripts/` (if the directory doesn't exist, it will need to be created). To include the submodule run `git submodule add https://github.com/nhsuk/ci-deployment.git scripts/ci-deployment`
+1. Create an executable `scripts/deploy.sh` file with the following contents:
+```
+#!/bin/sh
+
+git clone https://github.com/nhsuk/ci-deployment.git scripts/ci-deployment
+bash ./scripts/ci-deployment/deploy.sh
+```
 2. Add a `docker-compose.yml` and optionally, a `rancher-compose.yml` file to `./rancher-config/`, located in the root of the project
 3. **(Travis only)** Update the target repo's `.travis.yml` to include the sections included within the example `.travis.yml` in this repo
```

Comment:
I know this hasn't changed but linking `.travis.yml` to the one in the travis dir would help here (now there is a `.travis.yml` in the root of the repo.